docs/README.md

```diff
@@ -4,34 +4,38 @@ legacyUrl: https://docs.ipfs.io/
 description: The home page for developer documentation for IPFS, the InterPlanetary File System.
 ---
 
-# IPFS Documentation
+# Educate yourself
 
-**Welcome to IPFS!** We're glad you're here, and we hope this documentation site will help you make the decentralized Internet a better place for humanity. Whether you’re just learning about IPFS or are looking for detailed reference information, here's where to start.
+<p style="font-size:1.65rem; font-weight: 100; line-height: 2rem; margin-top: -15px;">Whether you're a tiny startup or a huge multinational company, you need to learn how to protect yourself and your users from the dangers of a centralized internet. This website contains everything you need to start using the InterPlanetary File System and help future-proof yourself.</p>
 
-IPFS docs are organized into the following main categories:
+## Host a static website on IPFS
```

Comment:
```suggestion
## Host your website on IPFS
```
"Your" makes it a little more personal. We may want to downplay the "static" aspect in the header, too.